tests/suite/config/array/computations-share.js

```diff
@@ -0,0 +1,84 @@
+var baseComputations = require('../../computations.js');
+
+baseComputations.mpcInterpreter = 'MPC';
+baseComputations.openInterpreter = 'OPEN';
+
+baseComputations.shareParameters = function (jiff_instance, test, testInputs) {
+  var numbers = testInputs['numbers'];
+
+  var input = numbers[jiff_instance.id]; // will be ignored if party is not a sender
+  var threshold = testInputs['threshold'];
+  var receivers = testInputs['receivers'];
+  var senders = testInputs['senders'];
+
+  return {
+    input: input,
+    threshold: threshold,
+    receivers: receivers,
+    senders: senders
+  }
+};
+
+baseComputations.shareHook = function (jiff_instance, test, testInputs, input, threshold, receivers, senders) {
+  var id = jiff_instance.id;
+  if (receivers.indexOf(id) === -1 && senders.indexOf(id) === -1) {
+    return null;
+  }
+
+  console.log(input);
+  return jiff_instance.share_ND_array(input, null,threshold, receivers, senders);
+};
+
+baseComputations.singleCompute = function (jiff_instance, shareParameters, test, values, interpreter) {
+  if (interpreter === baseComputations.openInterpreter) {
+    return values['numbers'][jiff_instance.id];
+  }
+
+  // MPC interpreter
+  // If we got here, we must be either a sender or a receiver
+
+  // Case 1: did not receive any shares, must have sent one, will receive an open.
+  if (shareParameters.receivers.indexOf(jiff_instance.id) === -1) {
+    console.log('did not receive any shares:', shareParameters)
+    return jiff_instance.receive_open(shareParameters.receivers, shareParameters.threshold);
```

Comment:
This should be `jiff_instance.receive_nd_open()` as jiff needs to know it's expecting an array not just a single share